README.md

```diff
@@ -97,6 +97,26 @@ make
 Note: by using the `-G` option of `cmake` you can specify a generator, e.g. `cmake -G Xcode -DCMAKE_BUILD_TYPE=Release ..` will generate an Xcode project.
 Please check `cmake --help` for more options.
 
+### Especially macOS
+
+```bash
+git clone --recursive https://github.com/Return-To-The-Roots/s25client s25client
+cd s25client
+mkdir -p build && cd build
+cmake .. \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_OSX_ARCHITECTURES=arm64 \
+  -DCMAKE_PREFIX_PATH="/opt/homebrew" \
+  -DENABLE_TESTS=OFF \
+  -DCMAKE_CXX_FLAGS="-std=c++17 \
+  -Wno-error=missing-noreturn \
+  -Wno-error=deprecated-copy \
+  -Wno-error=unused-parameter \
+  -Wno-error=undef \
+  -Wno-error=cast-qual" ..
```

Comment:
Do you really need that? If so we should fix it in the code so they are not required.

Can you use `-DRTTR_ENABLE_WERROR=OFF` and post the list of those warnings please?